app/Models/ConversionDefinition.php

```diff
@@ -22,28 +22,27 @@
 
 namespace Adshares\Adserver\Models;
 
-use Adshares\Adserver\Models\Traits\Ownership;
 use Adshares\Common\Domain\ValueObject\SecureUrl;
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Eloquent\Relations\BelongsTo;
 use Illuminate\Database\Eloquent\Relations\HasMany;
-use Illuminate\Database\Eloquent\SoftDeletes;
 use function route;
 
 class ConversionDefinition extends Model
 {
-    use Ownership;
-    use SoftDeletes;
-
     private const IN_BUDGET = 'in_budget';
     private const OUT_OF_BUDGET = 'out_of_budget';
```

Comment:
These two constants are not used explicitly in code. They are used in validation but not referrenced by `self::`

Reply:
It will be used later but I have also changed the rules section to use them explicitly.